### DISCUSSION TITLE
Toggle server/client network calls for development?

### DISCUSSION THREAD

**Discussion by BryceBlankinship** [remix-run/remix#7050] [Proposals]
I don't even know if this is possible, but I just wish I could debug my network calls like I do in any other application, by using Chrome Devtools. Can't do that when the calls are made on the server (I get that its the whole point of Remix, but I wish there was a config option to disable this for testing then re-enable). It seems foolish to me to re-create the wheel in terms of devtooling when Chrome Devtools is the best at what it does. Perhaps I need to explore other ways of testing my APIs, and its not so much a Remix problem but a me problem lol

**Comment by sergiodxa** [comment]
You can [`remix dev --debug`](https://remix.run/docs/en/main/other-api/dev#remix-dev---debug) to to use Node Inspector which lets you connect Chrome DevTools to inspect a Node process, including checking requests.

**Reply by BryceBlankinship** [reply]
all that really gives you access to is the console though... I was looking more for the network tab features
